vendor/modules.txt

```diff
@@ -1,2 +1,4 @@
+# github.com/aaronland/go-string v0.1.0
```

Comment:
Does this need to be cleaned up?

Reply:
See above.

Reply:
Why is this one commented out, but the next one isn't? Is that a Go convention I'm not familiar with?

Reply:
I believe the `go mod` command did that so...